compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       # needed for normal image
       #- ./assume-db:/var/lib/postgresql/data
       # /home/postgres/data is path for timescaledev image
-      - ./assume-db:/home/postgres/pgdata
+      - ./assume-db:/home/postgres/data
```

Comment:
It was the only way it worked on the extrernal machine, but I'll change it back

Reply:
I think you sometimes need to create the folder `mkdir assume-db` yourself, with user permission before changing this, because the timescaledb container behaves weirdly, I think..

Changing the path to something unused or commenting it out would lead to data being lost if the container is recreated - but other than that works of course as well